_pages/clients/python.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: article
-title: "Python"
+title: "Python Client Guide"
```

Comment:
Alright if we keep this as it is? Adding `Client Guide` is redundant when seeing the page in the side nav:

<img width="277" alt="Screen Shot 2020-11-04 at 11 18 35 AM" src="https://user-images.githubusercontent.com/8980868/98158903-89259e80-1e90-11eb-8fcc-65281d8cd139.png">

cc @JamesHoover

Reply:
Yeah thats fine to remove